### PR TITLE
Update to polkadot-v0.9.10 & some directory changes

### PR DESCRIPTION
**Pull request by irsal**
Note: Branch name is misleading, we'll do a parachain refactor in another PR. This was to get setup for the refactor.

Ensure the following are working post-upgrade:

- [x] 1. Build errors removed
- [x] 2. Tests
- [x] 3. Quadratic funding pallet working
- [x] 4. blocks are being produced by the standalone
- [x] 5. Testnet is functioning (please backup data first before upgrading testnet)

### CODE REVIEW

File: node/cli/src/chain_spec/turing.rs [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+// 		Some(properties),
+// 		// Extensions
+// 		Extensions {
+// 			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
+// 			para_id: id.into(),
+// 		},
+// 	)
+// }
+
+// fn testnet_genesis(
```
Comment (chrisli30) [inline node/cli/src/chain_spec/turing.rs:192]: Does this chain_spec/turing.rs need to be cleaned up? Do we still need those commented code?

File: node/cli/src/command.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-// This file is part of Substrate.
+// This file is part of OAK Network.
 
-// Copyright (C) 2017-2021 Parity Technologies (UK) Ltd.
+// Copyright (C) 2017-2021 Forge Labs, Inc.
```
Comment (chrisli30) [inline node/cli/src/command.rs:3]: Let's use "Chaintelligence Technology Inc." at all places for the Copyright. If you need explanation ping me offline.

**Review by chrisli30 — APPROVED** [review]
Approved. There are 2 non-blocking comments.

**Review by imstar15 — APPROVED** [review]

File: node/cli/src/chain_spec/turing.rs (hunk 1 above)
Comment (irsal) [reply node/cli/src/chain_spec/turing.rs:192]: I plan on cleaning it up in the next PR with parachain refactor

File: node/cli/src/chain_spec/oak_testnet.rs [hunk 3]
```diff
@@ -257,19 +253,19 @@ pub fn testnet_genesis(
 	const STASH: Balance = ENDOWMENT / 1000;
 
 	GenesisConfig {
-		frame_system: SystemConfig {
```
Comment (irsal) [inline node/cli/src/chain_spec/oak_testnet.rs:260]: might need to rename in oak-testnet.json

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -504,9 +521,14 @@ parameter_types! {
 	pub const SignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
 	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
 
-	// fallback: no need to do on-chain phragmen initially.
-	pub const Fallback: pallet_election_provider_multi_phase::FallbackStrategy =
-		pallet_election_provider_multi_phase::FallbackStrategy::OnChain;
```
Comment (irsal) [inline runtime/src/lib.rs:509]: regression for our npos

File: runtime/src/lib.rs [hunk 5]
```diff
@@ -338,6 +346,7 @@ impl pallet_babe::Config for Runtime {
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
 	type EpochChangeTrigger = pallet_babe::ExternalTrigger;
+	type DisabledValidators = Session;
```
Comment (irsal) [inline runtime/src/lib.rs:349]: consider rm'ing

**Pull request merged**